modules/ROOT/pages/bat-bdd-reference.adoc

```diff
@@ -84,6 +84,70 @@ describe `Demo scenario` in [
 
 A test suite must contain at least one test (.dwl) file.
 
+== Keeping Confidential Information Secure in Tests
+
+In your tests, you can access shared secrets in Anypoint Secrets Manager, which allows you to store securely sensitive information, such as a password, authentication token, endpoint URL, webhook URL, or something else. You create aliases for your shared secrets with the `bat grant` command. The BAT CLI stores aliases in your `bat.yaml` files in your test suites. You place aliases in your tests and, at runtime, the BAT CLI looks up the shared secrets by means of the aliases.
+
+[IMPORTANT]
+====
+In a given test suite, you can use aliases for shared secrets only if you plan to run the tests in that suite from a private location.
+====
+
```

Comment:
I am sure you have thought of this but in performing due diligence, I have to ask: is there any way to reduce the following steps to just one user action apiece, with less narrative underpinning?

Reply:
Also, I believe that the coding for substeps in AsciiDoc is ". ." rather than the letters you have used here.

Reply:
GLOBALLY break multisentence steps into a new para after the first sentence.

Reply:
1. I don't see that extra info as narration. I see it as necessary because of the complexity of the task.
2. No, I'm specifying in Asciidoc that I want to use the lowercase letters at the second level, not integers again.
3. Resolved here.